hermes/file_gen.go

```diff
@@ -1,40 +1,33 @@
 package hermes
 
-
 import (
 	"fmt"
-	"strconv"
 	"crypto/sha1"
 )
 
 type HermesFile struct {
 	name string
 	contents string
 }
-func (file *HermesFile) GenerateFileName(file_ID int, file_checksum string) {
-	id_string := ""
-	if file_ID < 10 {
-		id_string = "0" + strconv.Itoa(file_ID)
-	} else {
-		id_string = strconv.Itoa(file_ID)
-	}
-	file.name := "Hermes_"+id_string+file_checksum
+func (file *HermesFile) generateFileName(file_ID int, file_checksum string) {
```

Comment:
Use snake_case for variable names.
```suggestion
func (file *HermesFile) generateFileName(file_id int, file_checksum string) {
```